### DISCUSSION TITLE
Is a new server necessary?

### DISCUSSION THREAD

**Discussion by MartinJindra** [docker-mailserver/docker-mailserver#2923] [Q&A]
# Is a new server necessary?

Other

## Description

I have a main production server where I want to run `docker-mailserver` on. Many similar project advice to deploy the email services on an unused server. Is this also necessary here?

---

MAINTAINER EDIT: This was transferred into a discussion from an issue.

**Comment by georglauterbach** [comment]
I do not see a pressing reason to do so _unless_ you are doing someting extremly security critical. I havy my mail server colocated in the very same server that my other infrastructure runs on. This is the benefit of containers: everything works well together even on the same system.

My advice: there is not need to get a new server. This would probably be a waste of resources.
